### ISSUE TITLE
[feature] Pasting Raw Text Repeatedly Should Not Fully Cover Up Previous Pastes

### ISSUE THREAD

**Issue by seflless** [tldraw/tldraw#401]
See video. To reproduce:
 1. Selected some text from a text editor
 2. I did Command+V 8 times
 3. 8 text shapes were created on top of each other
 4. This makes it easy to not realize you've got duplicates on your board

https://user-images.githubusercontent.com/1845/143690995-00e549af-8d7c-4b2a-8e8a-82619a2b7249.mov

**Comment by seflless** [comment]
This is a non issue when copy/pasting text shapes (versus raw text

https://user-images.githubusercontent.com/1845/143691058-473c6ff7-f6ba-45e3-a9e3-a4d2562f2ec3.mov

)

**Comment by steveruizok** [comment]
Good catch! (I think there's also a bug with "regular" paste that doesn't offset the second pasted object, only the third and there on).
